Header header
bool fly_flying
uint32 frame_count
float32 value
string muscle
